docs/design/tdm-plugin.md

```diff
@@ -0,0 +1,41 @@
+# Time Division Multiplexing (TDM) plugin
+
+## Introduction
+
+In a co-located environment, some nodes are in both Kubernetes cluster and Yarn cluster. For these nodes, Kubernetes and Yarn cluster can use these resource by time-sharing multiplexing.
+For example, in 8:00~24:00 the on-line service in kubernetes can can use these nodes. And in 0:00~8:00, kubernetes need evict the workload on these nodes, left the resource for Yarn cluster.
+
+
+## Solution
+
+Frist, we need mark out these nodes(by add `volcano.sh/revocable-zone` for nodes) which are in both Kubernetes cluster and Yarn cluster. We call these node are `revocable node`. For the kubernetes 
+workload, we add `volcano.sh/preemptable: "true"` annotation for pod/podgroup to distinguish which pod can be dispatched to the `revocable node`. We call these pod are `preemptable task`.
+TDM plugin enhance the volcano time-sharing multiplexing resource ability. It will try to dispatch `preemptable task` to `revocable node` in node revocable time and evict the `preemptable task` from 
+`revocable node` out of revocable time.
+
+![tdmsolution](./images/tdmsolution.png)
+
+1. Add `volcano.sh/preemptable` annotaion for volcano job. Pod/PodGroup will inherit this annotation. The pod with `volcano.sh/preemptable: "true"` annotation can be dispatched to `revocable node`.
```

Comment:
this should no bind to volcano job; other Kind can add this annotation to `PodTemplate`.